Utilities/python/ConfigureJobs.py

```diff
@@ -181,35 +181,25 @@ def getListOfHDFSFiles(file_path):
 
 # TODO: Would be good to switch the order of the last two arguments
 # completely deprecate manager_path without breaking things
-def getListOfFiles(filelist, selection, manager_path="", analysis=""):
-    if manager_path is "":
-        manager_path = getManagerPath()
+def getListOfFiles(filelist, analysis, selection=""):
+    manager_path = getManagerPath()
     data_path = "%s/%s/FileInfo" % (manager_path, getManagerName())
-    group_path = "%s/AnalysisDatasetManager/PlotGroups" % manager_path
+    group_path = "%s/%s/PlotGroups" % (manager_path, getManagerName())
     data_info = UserInput.readAllInfo("/".join([data_path, "data/*"]))
     mc_info = UserInput.readAllInfo("/".join([data_path, "montecarlo/*"]))
-    analysis_info = UserInput.readInfo("/".join([data_path, analysis, selection])) \
-        if analysis != "" else []
-    valid_names = (data_info.keys() + mc_info.keys()) if not analysis_info else analysis_info.keys()
-    group_names = UserInput.readAllInfo("%s/%s.py" %(group_path, analysis)) if analysis else dict()
+    
+    valid_names = UserInput.readInfo("/".join([data_path, analysis, selection])) if selection \
+        else (data_info.keys() + mc_info.keys())
+    group_names = UserInput.readAllInfo("%s/%s.py" %(group_path, analysis))
     names = []
     for name in filelist:
         if ".root" in name:
             names.append(name)
         # Allow negative contributions
         elif name[0] == "-" :
             names.append(name)
-        elif "WZxsec2016" in name:
-            dataset_file = manager_path + \
-                "%s/FileInfo/WZxsec2016/%s.json" % (getManagerPath(), selection)
-            allnames = json.load(open(dataset_file)).keys()
-            if "nodata" in name:
-                nodata = [x for x in allnames if "data" not in x]
-                names += nodata
-            elif "data" in name:
-                names += [x for x in allnames if "data" in x]
-            else:
-                names += allnames
+        elif "*" in name:
+            names += fnmatch.filter(valid_names, name)
         elif name in group_names:
             names += group_names[name]['Members']
         elif "*" in name:
```

Comment:
this same elif condition appears twice, something went wrong in the merge (I think we each changed things)